update verify message page to new UI design

oshorefueled opened the pull request:
This PR resolves issue #281

oshorefueled committed:
update verify message with new ui design

* add sub page layout
* add info modal template

oshorefueled committed:
update input error behaviour with new UI design

oshorefueled committed:
correct call to event handle method

oshorefueled committed:
highlight editor border lines on error

oshorefueled committed:
add page description

oshorefueled committed:
remove assigning to error label

oshorefueled committed:
restore validation on address input change

oshorefueled committed:
remove direct assigning of input error

Sirmorrison commented on ui/page.go:
```diff
@@ -630,6 +639,72 @@ func (page pageCommon) SelectedAccountLayout(gtx layout.Context) layout.Dimensio
+		sp.back()
+	}
+
+	return decredmaterial.Card{Color: page.theme.Color.Surface, Rounded: true}.Layout(gtx, func(gtx C) D {
```
kindly rebase as the Card widget has been updated. `Color` and `Rounded` params no longer exist in card struct.

Sirmorrison reviewed (changes requested):
* A little extra padding around the modal will be nice.
<img width="461" alt="image" src="https://user-images.githubusercontent.com/27733432/103338421-b0f34600-4a7e-11eb-85ed-e73dbc108fe7.png">

The valid and invalid text colors does not align with the mockup design.
<img width="261" alt="image" src="https://user-images.githubusercontent.com/27733432/103338528-f9126880-4a7e-11eb-9c75-a2dae2f81f93.png">

* The gray text background is way darker than the UI design. you can use the gray color used on the transaction details page.
<img width="234" alt="image" src="https://user-images.githubusercontent.com/27733432/103338608-3a0a7d00-4a7f-11eb-977e-655cc2c65cef.png">

tx details page..
<img width="306" alt="image" src="https://user-images.githubusercontent.com/27733432/103338664-6625fe00-4a7f-11eb-9717-b9238ddf588b.png">

* The color used for the clear all button is gray as againt the the blue used here.
<img width="234" alt="image" src="https://user-images.githubusercontent.com/27733432/103338817-e9475400-4a7f-11eb-8a6e-521f1c80a5ca.png">

<img width="103" alt="image" src="https://user-images.githubusercontent.com/27733432/103338949-5d81f780-4a80-11eb-83f8-5d4882373840.png">

oshorefueled committed:
correct colour inaccuracies

oshorefueled committed:
add more padding to modals

Sirmorrison commented on ui/page.go:
```diff
@@ -669,6 +678,72 @@ func (page pageCommon) SelectedAccountLayout(gtx layout.Context) layout.Dimensio
+				cancelText: "Got it",
+			}
+		}()
+	}
```
From what i understand about this function, its supposed to used by all the subpages. However whenever the `subPageInfoButton` info button is clicked, it will only render the verify message modal template. I believe this should be made flexible.

Sirmorrison commented on ui/modal_templates.go:
```diff
@@ -275,6 +296,12 @@ func (m *ModalTemplate) handle(th *decredmaterial.Theme, load *modalLoad) (templ
+			load.cancel.(func())()
+		}
+		template = m.verifyMessageInfo()
+		return
```
This `InfoTemplate` is supposed to render for all the information on the pages i presume. currently, this renders only the `verifyMessageInfo`. We can either make this dynamic so it works with all info on the pages or have the `InfoTemplate` renamed to `verifyMsgInfoTemplate`.

oshorefueled replied:
you're correct. My intention was to do that before pushing, I'll do that on my next commit.

oshorefueled committed:
make subpage layout usable by all subpages

make subpage layout usable by all subpages

assign a more descriptive ID to info template

Sirmorrison reviewed (approved):
PR looks good.
